This patch is intended to fix `genrb' with internal error during the build
(probably after build as well). Originated from Debian.

This patch can be used in any architecture.

--- icu/source/config/mh-linux  2015-03-27 21:11:20.000000000 +0000
+++ icu-changed/source/config/mh-linux  2015-08-17 01:26:15.000000000 +0000
@@ -23,7 +23,7 @@
 LD_RPATH_PRE = -Wl,-rpath,
 
 ## These are the library specific LDFLAGS
-LDFLAGSICUDT=-nodefaultlibs -nostdlib
+LDFLAGSICUDT=
 
 ## Compiler switch to embed a library name
 # The initial tab in the next line is to prevent icu-config from reading it.
